From f2c2acef3861b2dafc84a23c90ff2b68eb713616 Mon Sep 17 00:00:00 2001
From: Benjamin Feller <benjamin.feller@viveris.fr>
Date: Mon, 11 Sep 2023 15:52:48 +0200
Subject: [PATCH] fix rohc_buf_* export preventing compilation in kernel module
 mode

---
 linux/kmod.c | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/linux/kmod.c b/linux/kmod.c
index f2bf2b6a..5ea5b476 100644
--- a/linux/kmod.c
+++ b/linux/kmod.c
@@ -59,18 +59,6 @@ EXPORT_SYMBOL_GPL(rohc_packet_is_ir);
 EXPORT_SYMBOL_GPL(rohc_packet_carry_static_info);
 EXPORT_SYMBOL_GPL(rohc_packet_carry_crc_7_or_8);
 
-EXPORT_SYMBOL_GPL(rohc_buf_is_malformed);
-EXPORT_SYMBOL_GPL(rohc_buf_is_empty);
-EXPORT_SYMBOL_GPL(rohc_buf_push);
-EXPORT_SYMBOL_GPL(rohc_buf_pull);
-EXPORT_SYMBOL_GPL(rohc_buf_avail_len);
-EXPORT_SYMBOL_GPL(rohc_buf_data_at);
-EXPORT_SYMBOL_GPL(rohc_buf_data);
-EXPORT_SYMBOL_GPL(rohc_buf_prepend);
-EXPORT_SYMBOL_GPL(rohc_buf_append);
-EXPORT_SYMBOL_GPL(rohc_buf_append_buf);
-EXPORT_SYMBOL_GPL(rohc_buf_reset);
-
 
 /*
  * Compression API
-- 
2.34.1

